openreview/tools.py

```diff
@@ -345,13 +367,13 @@ def get_all_tags(client, invitation, limit=1000):
     '''
     Given an invitation, returns all Tags that respond to it, ignoring API limit.
     '''
-    return _get_all(client.get_tags, invitation, limit)
+    return [t for t in iterget(client.get_tags, invitation=invitation, limit=limit)]
```

Comment:
can you do?

```
return list(iterget(client.get_tags, invitation=invitation, limit=limit))
```

Based on:
https://stackoverflow.com/questions/3790848/fastest-way-to-convert-an-iterator-to-a-list

Reply:
oh! I didn't realize you could do that. i will change it